src/test/java/CompoundCommandCopyTest.java

```diff
@@ -1,7 +1,7 @@
-import edu.kis.powp.jobs2d.command.CompoundCommand;
-import edu.kis.powp.jobs2d.command.DriverCommand;
-import edu.kis.powp.jobs2d.command.OperateToCommand;
-import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.command.driverCommands.ICompoundCommands.CompoundCommand;
```

Comment:
I am not sure if creating new packages with such long names is necessary. Convention looks invalid (uppercase). Maybe resign from such refactoring.

Reply:
done